### DISCUSSION TITLE
🎉 Payload 2.0 Released

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#3501] [Announcement]
## We're happy to announce that Payload 2.0 has been released!

The [blog post](https://payloadcms.com/blog/payload-2-0) with full details and video walkthroughs.

Also have a look at the [release notes](https://github.com/payloadcms/payload/releases/tag/v2.0.0) if you are upgrading from v1.

![image](https://github.com/payloadcms/payload/assets/65888/3697da65-f93f-4860-ab91-aaf3ae79735e)

**Comment by hunghvu** [comment]
Congrats! Will look into migrating my company production project to v2.0!

**Comment by balmacefa** [comment]
Thank you for providing an update!

I've had issues migrating from v.1 to v.1.x in the past; payload cms updates usually stabilise around 2/3 weeks; have you considered a release candidate flow before to uncover bugs and enhance the updating guides?

**Reply by DanRibbens** [reply]
We were doing beta releases and working furiously last week upgrade templates and plugins which were public betas, we just didn't broadcast it. Now that we are situated with our monorepo we are in a much better place to iterate on releases and are talking about how we will use tags for betas or another way to automate this in the future.

Good feedback, we're going to continue to improve release management!

**Reply by alialali313** [reply]
G

**Comment by grundiss** [comment]
Hey team!
Great work, thank you :) 
I have 2 questions:
1. I wonder for how long you are planning to support 1.x?
2. Is there an official 1x-to-2.0 migration guide? Like moving stuff from mongo to postgress and from Slate to Lexical?

**Reply by STR1234** [reply]
Yes I am with @grundiss and especially interested in the topic of moving/migrating Slate content to Lexical - that would be awesome. 👍

**Reply by jacobsfletch** [reply]
We will continue to support 1.x for security fixes only, as necessary. RE: Slate->Lexical, roping in @AlessioGr here. He's been working on making this as painless as possible. For general help migrating from 1.x->2.x, you can reference our [Examples Directory](https://github.com/payloadcms/payload/tree/main/examples) where most of the examples have been migrated over to 2.0 already, as well as our [Templates Directory](https://github.com/payloadcms/payload/tree/main/templates). [Here's a pretty clean PR](https://github.com/payloadcms/payload/pull/3506) showing all the changes made to the [Auth Example](https://github.com/payloadcms/payload/tree/main/examples/auth).

**Reply by AlessioGr** [reply]
Migration from Slate to Lexical is SUPER simple! Here's a video showcasing it:

https://github.com/payloadcms/payload/assets/70709113/2ec8a3fd-5559-438b-b3cf-a4a5f9486bdb

Simply
1. Change the field definition from Slate to Lexical (by changing the `editor` property)
2. Now, obviously, the lexical editor sits on top of Slate data, which it cannot read... Oh wait, it actually can! Just add the `SlateToLexicalFeature()` to the lexical editor, and done! 

Now, if you are happy with the conversion, you can save the document (so it overrides the Slate data with the converted Lexical data) and remove the `SlateToLexicalFeature` (if you want. you don't have to).

If you have any custom Slate nodes added, you can simply add your own converter for those nodes as props to the SlateToLexicalFeature.

This is all very new and not documented yet - documentation will follow soon

**Reply by STR1234** [reply]
Thank you so much @AlessioGr :) Amazing stuff and thanks for the video and your effort here - so hyped to try Lexical 👍

**Reply by STR1234** [reply]
Thanks @jacobsfletch - looking forward to the migration of my projects :)

**Reply by ChristopherNowlan** [reply]
@STR1234 @AlessioGr doesn't look like Lexical works with the plugin-form-builder plugin :(

**Comment by tylerwatrich** [comment]
Very excited about this! The new lexical editor looks very promising!

**Comment by alialali313** [comment]
H

**Comment by tiagomarin** [comment]
How could I start the project already on 2.0? Or I always need to install 1.0 and then upgrade it manually?
It seems that `npx create-payload-app@latest` creates a 1.0 project.
